tools/setup.py

```diff
@@ -56,19 +58,28 @@ def check_directory(self, name):
                                   + "https://github.com/SCE-Development/"
                                   + name, stderr=subprocess.STDOUT, shell=True)
 
-    def check_mongo(self):
+    def check_docker(self):
         """
-        This method checks for mongo installation
+        This method checks for docker installation and
+        if it is running
         """
-        self.check_installation("mongo", "mongo --version",
-                                    "https://www.mongodb.com/"
-                                    + "try/download/community")
+        docker_status = check_docker_status()
+        if not docker_status['is_installed']:
+            self.color.print_red('Docker not found')
```

Comment:
This seems to be doing the same thing as [`tools/setup.py:39`](https://github.com/SCE-Development/SCE-CLI/blob/a43bfa2548605d22d3fe9c65462b7db952e2bad0/tools/setup.py#L39). Maybe abstract the block of code to a common function like

```py
def prompt_user_to_install(self, name, url):
  self.color.print_red(name + " not found")
  print("visit here to install: ")
  self.color.print_purple(link)
  input("press enter to continue: ")
```